src/term/mod.rs

```diff
@@ -377,7 +398,7 @@ impl Term {
         self.next_title.take()
     }
 
-    pub fn new(size: SizeInfo) -> Term {
+    pub fn new(size: SizeInfo, config : &Config) -> Term {
```

Comment:
Please swap the order of arguments here. We are trying to keep `Config` consistently as the first argument wherever it's used.